src/components/Main/Main.js

```diff
@@ -7,13 +7,42 @@ import Tutorial from "../Tutorial/Tutorial";
 import { levels } from "../../components/levels";
 import { maxLevel, sticker_names } from "../../constants/constants";
 import "./Main.css";
+import Joyride from 'react-joyride';
 
 class Main extends React.Component {
   state = {
     open: false,
     stickerStyles: {},
     solved: false,
     showTutorial: false,
+    run: true,
+    steps: [
+      {
+        target: ".level-tutorial-button",
+        content: "Feeling stuck? Here are some tips!",
+        disableBeacon: "true",
+      },
+      {
+        target: ".Level-sidebar",
+        content: "Click here to navigate to a different level!",
+        disableBeacon: "true",
+      },
+      {
+        target: "path",
+        content: "Move on to the next level!",
```

Comment:
Same as before, I might change this to 'Click here to move on to the next level!'